### PR TITLE
Fix #918 - Governance Act errors in DOM tests

### LINKED ISSUE #918: Fix Governance error messages during DOM tests
image

### PR DESCRIPTION
**Pull request by abefernan**
Closes #918.

### CODE REVIEW

File: packages/sil-governance/src/routes/show-electorate/index.dom.spec.ts [hunk 1]
```diff
@@ -37,7 +37,9 @@ describe("DOM > Feature > Show Electorate", () => {
       },
     });
 
-    electorateDom = await travelToShowElectorate(store, "1");
+    await act((async () => {
+      electorateDom = await travelToShowElectorate(store, "1");
+    }) as () => void);
   }, 60000);
```
Comment (albertandrejev) [inline packages/sil-governance/src/routes/show-electorate/index.dom.spec.ts]: And here if possible.

File: packages/sil-governance/src/routes/show-electorate/index.dom.spec.ts [hunk 2]
```diff
@@ -93,7 +95,10 @@ describe("DOM > Feature > Show Electorate", () => {
...
 
   it("has an Equal Electorate with expected members", async () => {
-    electorateDom = await travelToShowElectorate(store, "1");
+    await act((async () => {
+      electorateDom = await travelToShowElectorate(store, "1");
+    }) as () => void);
+
```
Comment (albertandrejev) [inline packages/sil-governance/src/routes/show-electorate/index.dom.spec.ts]: here

File: packages/sil-governance/src/routes/show-electorate/index.dom.spec.ts [hunk 3]
```diff
@@ -110,7 +115,10 @@ describe("DOM > Feature > Show Electorate", () => {
   }, 60000);
 
   it("has a Weighted Electorate with expected members", async () => {
-    electorateDom = await travelToShowElectorate(store, "2");
+    await act((async () => {
+      electorateDom = await travelToShowElectorate(store, "2");
+    }) as () => void);
```
Comment (albertandrejev) [inline packages/sil-governance/src/routes/show-electorate/index.dom.spec.ts]: here

File: packages/sil-governance/src/routes/dashboard/index.dom.spec.ts [hunk 4]
```diff
@@ -39,19 +43,18 @@ describe("DOM > Feature > Dashboard", () => {
...
       blockchain: {
         lastBlockTime: new ReadonlyDate(),
         lastBlockHeight: 44447774,
       },
     });
 
-    dashboardDom = await travelToDashboard(store);
+    await act((async () => {
+      dashboardDom = await travelToDashboard(store);
+    }) as () => void);
```
[the remark this answers is not in the document]
Comment (abefernan) [reply packages/sil-governance/src/routes/dashboard/index.dom.spec.ts]: I tried extracting it exactly like that but I got an error having to do with race conditions. I can tinker with it a bit more.

[the remark this answers is not in the document]
Comment (abefernan) [reply packages/sil-governance/src/routes/dashboard/index.dom.spec.ts]: I managed to extract it :100:

**Pull request merged**